src/test/java/core/basesyntax/services/ReportServiceTest.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax.services;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ReportServiceTest {
+    private static final String FIRST_LINE = "fruit,quantity";
+    private static ReportService reportService;
+    private Map<String, Integer> storage;
+
+    @BeforeAll
+    static void beforeAll() {
+        reportService = new ReportServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        storage = new LinkedHashMap<>();
+    }
+
+    @Test
+    void testGenerate_reportWithValidData_OK() {
+        storage.put("apple", 10);
+        storage.put("banana", 5);

```

Comment:
I think creating constants with fruits and their quantities will increase code readability